### PR TITLE
Operator post-merge: update the data/versions.yaml

### PR DESCRIPTION
**Pull request by elfinhe**
Update the data/versions.yaml, and also update the built-in link to istio/istio repo.

### CODE REVIEW

File: operator/cmd/mesh/manifest-versions.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	versionsMapURL = "https://raw.githubusercontent.com/istio/operator/master/data/versions.yaml"
+	versionsMapURL = "https://raw.githubusercontent.com/istio/istio/master/operator/data/versions.yaml"
```
Comment (howardjohn) [inline operator/cmd/mesh/manifest-versions.go]: Woah.... we are pulling this from github??? this is not a good idea

Comment (elfinhe) [reply operator/cmd/mesh/manifest-versions.go]: I agree, what's your suggestion? 

make it as a built-in VFS file?

Comment (howardjohn) [reply operator/cmd/mesh/manifest-versions.go]: I don't think anything should be in VFS presonally, but while we have everything else in VFS then this should be as well?

**Review by howardjohn — APPROVED** [review]

**Pull request merged**